### ISSUE TITLE
DeleteEntriesAsync raises error when issued in a batch

### ISSUE THREAD

**Issue by HEBOS** [object/Simple.OData.Client#376]
I am trying to delete many records using batch (in order to avoid many single REST requests).

This is the code:

```
batch += async c => await c
                    .For<CachedMessage>()
                        .Filter(x => x.isSynced)
                        .DeleteEntriesAsync();

                // Perform batch update
                await batch.ExecuteAsync().ConfigureAwait(false);
```

This is the exception: "An error occurred while sending the request."
If I try to issue delete without batch, it works as expected.

**Comment by object** [comment]
The reason for its failure may be the same as in #375. I wrote a comment there, will investigate more.

**Comment by object** [comment]
I believe this should be fixed in version 4.26.0. Please try it out.
